### PR TITLE
Remove unnecessary self handling of TestBot

### PR DESCRIPTION
**Pull request by nvzard**
Closes https://github.com/coala/corobo/issues/529

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: tests/coatils_test.py [hunk 1]
```diff
@@ -7,110 +8,87 @@
...
-                         'At line 3')
-        self.assertEqual(Coatils.position(1, None, 3, 6),
-                         'Between line 1 and position 3:6')
-        self.assertEqual(Coatils.position(1, 4, 5, None),
-                         'Between position 1:4 and line 5')
+extra_plugin_dir = 'plugins'
+
+
+@vcr.use_cassette('tests/cassettes/coatils_total_bears.yaml')
+def test_total_bears():
```
Comment (jayvdb) [inline tests/coatils_test.py]: See https://github.com/coala/corobo/issues/529#issuecomment-392229810

File: tests/wolfram_alpha_test.py [hunk 2]
```diff
@@ -2,26 +2,31 @@
...
 
     @my_vcr.use_cassette('tests/cassettes/wa.yaml')
     def test_wa(self):
-        self.testbot.assertCommand("!wa 2^6", "64")
+        self.push_message('!wa 2^6')
+        self.assertIn('64', self.pop_message())
         with self.assertLogs() as cm:
-            self.testbot.assertCommand('!wa this is a sentence',
+            self.assertCommand('!wa this is a sentence',
                                        'Dunno')
```
Comment (meetmangukiya) [inline tests/wolfram_alpha_test.py]: match indentation

File: tests/wolfram_alpha_test.py [hunk 3]
```diff
@@ -2,26 +2,31 @@
...
-    def setUp(self):
-        self.testbot = TestBot(extra_plugin_dir='plugins',
-                               loglevel=logging.ERROR)
-        self.testbot.start()
+    def setUp(self,
+              extra_plugin_dir=None,
+              extra_test_file=None,
+              loglevel=logging.DEBUG,
+              extra_config=None):
+        super().setup(extra_plugin_dir='plugins',
```
Comment (meetmangukiya) [inline tests/wolfram_alpha_test.py]: Use `setUp` and remove next line. Similarly in other cases

Comment (nvzard) [reply tests/wolfram_alpha_test.py]: I should die :persevere:

Comment (meetmangukiya) [reply tests/wolfram_alpha_test.py]: It's okay, no need to die.

**Comment by meetmangukiya** [comment]
@gitmate-bot rebase

**Comment by meetmangukiya** [comment]
@gitmate-bot ff

**Pull request merged**